QKSMS/src/main/java/org/w3c/dom/smil/ElementExclusiveTimeContainer.java

```diff
@@ -31,7 +31,7 @@ public void setEndSync(String endSync)
                                      throws DOMException;
 
     /**
-     *  This should support another method to get the ordered collection of 
+     *  This should support another method to getConversation the ordered collection of
```

Comment:
Seems to be a lot of comments with the word "get" that got replaced with "getConversation"